backend/src/main/scala/cromwell/backend/standard/callcaching/StandardCacheHitCopyingActor.scala

```diff
@@ -154,11 +158,11 @@ abstract class StandardCacheHitCopyingActor(val standardParams: StandardCacheHit
     stay()
   }
 
-  private def lookupSourceCallRootPath(sourceJobDetritusFiles: Map[String, String]): Path = {
-    sourceJobDetritusFiles.get(JobPaths.CallRootPathKey).map(getPath).get recover {
-      case failure =>
-        throw new RuntimeException(s"${JobPaths.CallRootPathKey} wasn't found for call ${jobDescriptor.call.fullyQualifiedName}", failure)
-    } get
+  private def lookupSourceCallRootPath(sourceJobDetritusFiles: Map[String, String]): Try[Path] = {
```

Comment:
`getPath` does return a `Try[Path]` though already so it would be an `Option[Try[Path]]`